### DISCUSSION TITLE
Triangles aren't quite big enough when using powerline themes.

### DISCUSSION THREAD

**Discussion by jangrd** [i3/i3#5032] [Q&A] [answered]
The triangles are a bit to small and don't quite reach the top and bottom of the bar.
![image](https://user-images.githubusercontent.com/56997476/176008621-2bd11b36-f830-415a-a114-0bb5bf24db3f.png)
So far I had a few issues with fonts and ended up using DejaVu Sans Mono for Powerline.
I thought maybe the triangles are just rendered too far to the right, so I edited the .json and that wasn't the case.

**Comment by jangrd** [comment, marked as answer]
I am sorry, I made this in the wrong repo. Please delete it.

**Reply by Airblader** [reply]
It's the correct repository.
